File: gildedrose/src/main/java/edu/craptocraft/gildedrose/domain/ConjuredItem.java
package edu.craptocraft.gildedrose.domain;

public class ConjuredItem extends NormalItem {
    public ConjuredItem(String name, int sell_in, int quality){
        super(name, sell_in, quality);
    }

@Override 
public void updateQuality(){
    
    if(getSell_in() > 0){
        computeQuality(-2);
    }
    else{
        computeQuality(-4);
    }
    setSell_in();
    }
}

File: gildedRosePy/src/conjuredItem.py
from src.normal_item import *

class ConjuredItem(NormalItem):
    
    def __init__(self, name, sell_in, quality):
        NormalItem.__init__(self, name, sell_in, quality)
        
    def updateQuality(self):
        
        if self.sell_in > 0:
            self.setQuality(-2)
        else:
            self.setQuality(-4)
        self.setSell_in()
        